interface/src/app/greenelab/stats.spec.js

```diff
@@ -40,3 +40,83 @@ describe('Test the hypergeometric test function in stats.js', function() {
     }
   );
 });
+
+describe('tTest in stats.js', function() {
+  beforeEach(module('greenelab.stats'));
+
+  var MathFuncts;
+
+  beforeEach(inject(function(_MathFuncts_) {
+    MathFuncts = _MathFuncts_;
+  }));
+
+  it('should be defined', function() {
+    expect(MathFuncts).toBeDefined();
+    expect(MathFuncts).toBeTruthy();
+  });
+
+  // These tests compare results from tTest against results obtained from
+  // R's t.test
+  // http://stat.ethz.ch/R-manual/R-devel/library/stats/html/t.test.html
+  it('should produce correct results for several examples', function() {
+    var tt;
+
+    // Compare with R's output from:
+    // tt <- t.test(c(0.7, -1.6, -0.2, -1.2, -0.1, 3.4, 3.7, 0.8, 0.0, 2.0),
+    // c(1.9, 0.8, 1.1, 0.1, -0.1, 4.4, 5.5, 1.6, 4.6, 3.4))
+    tt = MathFuncts.tTest(
+      [0.7, -1.6, -0.2, -1.2, -0.1, 3.4, 3.7, 0.8, 0.0, 2.0],
+      [1.9, 0.8, 1.1, 0.1, -0.1, 4.4, 5.5, 1.6, 4.6, 3.4]
+    );
+    expect(+tt.testValue().toFixed(10)).toEqual(-1.8608134675);
```

Comment:
Cool! I will do it that way. @rzelayafavila, should I update the `expect`s from your hypergeometric test's test while I'm in there?